spec/features/till_features_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'till'
+
+feature 'As a customer ordering two lattes' do 
+  scenario 'I can get a receipt' do
+    till = Till.new
+    till.place_order "Cafe Latte"
+    till.place_order "Cafe Latte"
+    till.checkout
+    till.totals
+    expect(till.print_receipt).to eq [{"Cafe Latte"=>4.75}, {"Cafe Latte"=>4.75}, {"Tax"=>0.82}, {"Total"=>9.50}]
```

Comment:
Line is too long. [113/80]<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.